packages/manager/modules/pci/src/projects/project/users/roles-matrix/roles-matrix.html

```diff
@@ -0,0 +1,108 @@
+                    <span class="sr-only" data-translate="pci_projects_project_users_roles_matrix_customize"></span>
+                </button>
+                <oui-dropdown-content>
+                    <div class="oui-datagrid-parameters__body pt-1">
+                        <div
+                            class="oui-datagrid-parameters__column"
+                            data-ng-repeat="role in $ctrl.roles.roles|orderBy:name track by role.id">
+                            <div class="oui-datagrid-parameters__column-column">
+                                <oui-checkbox data-model="$ctrl.columns[role.id]">
+                                    <span data-ng-bind=":: role.name"></span>
+                                </oui-checkbox>
+                            </div>
+                        </div>
+                    </div>
+                </oui-dropdown-content>
+            </oui-dropdown>
+        </div>
+    </div>
+    <div class="users-roles-matrix oui-datagrid-responsive-container oui-datagrid-responsive-container_sticky-checkbox">
+        <div class="oui-datagrid-responsive-container__sticky-container oui-datagrid-responsive-container__sticky-container_scroll-begin oui-datagrid-responsive-container__sticky-container_scroll-end">
+            <div class="oui-datagrid-responsive-container__overflow-container">
+                <table class="oui-datagrid">
+                    <thead class="oui-datagrid__headers">
+                        <tr>
+                            <th class="oui-datagrid__header oui-datagrid__cell-sticky users-roles-matrix__service"></th>
+                            <th class="oui-datagrid__header oui-datagrid__cell-sticky users-roles-matrix__permission"></th>
+                            <th class="oui-datagrid__header text-center"
+                                data-ng-repeat="role in $ctrl.roles.roles|orderBy:name track by role.id"
+                                data-ng-bind="role.name"
+                                data-ng-if="$ctrl.columns[role.id]"></th>
+                        </tr>
+                    </thead>
+                    <tbody data-ng-repeat="service in $ctrl.roles.services track by service.name">
+                        <tr class="oui-datagrid__row"
+                            data-ng-repeat="permission in service.permissions track by permission.name">
+                            <td class="oui-datagrid__cell oui-datagrid__cell-sticky users-roles-matrix__service"
+                                data-ng-if="$first"
+                                rowspan="{{service.permissions.length}}">{{ service.name }}</td>
+                            <td class="oui-datagrid__cell oui-datagrid__cell-sticky users-roles-matrix__permission">{{ permission.name }}</td>
+
+                            <td class="text-center oui-datagrid__cell"
+                                data-ng-repeat="permissionRole in permission.roles track by permissionRole.id"
```

Comment:
```suggestion
                                data-ng-repeat="permissionRole in ::permission.roles track by permissionRole.id"
```
?

Reply:
No because values inside table won't be updated